### PR TITLE
feat: exponential backoff

### LINKED ISSUE #10: Exponential backoff for schema validations

### PR DESCRIPTION
**Pull request by maxlivinci**
resolves #10

### CODE REVIEW

File: services/validation/internal/service/validation_service.go [hunk 1]
```diff
@@ -68,6 +68,36 @@ func (svc *validationService) ValidateNode(node *node.Node) {
 	})
 }
 
+func (svc *validationService) readFromProfileURL(profileURL string) (interface{}, error) {
+	document := gojsonschema.NewReferenceLoader(profileURL)
+
+	var data interface{}
+
+	// TODO: Need a feature toggle mechanism.
+	if true {
```
Comment (maxlivinci) [inline services/validation/internal/service/validation_service.go:77]: @geoffturk Set to `false` to test exponential backoff.

We probably need a feature toggle mechanism to turn on/off this feature whenever we want.

File: charts/murmurations/charts/core/templates/ingress/ingress.yaml [hunk 2]
```diff
@@ -19,10 +19,12 @@ spec:
       secretName: murmurations-tech-tls
   {{- end }}
   rules:
-    {{- if eq .Values.global.env "development" }}
-    - host: index.murmurations.dev
-    {{- else }}
+    {{- if eq .Values.global.env "production" }}
+    - host: index.murmurations.com
```
Comment (geoffturk) [inline charts/murmurations/charts/core/templates/ingress/ingress.yaml:23]: Please change .com to .network for production

File: services/validation/internal/service/validation_service.go (hunk 1 above)
Comment (geoffturk) [reply services/validation/internal/service/validation_service.go:77]: OK. I made a note about it in the feature toggle issue.

**Review by geoffturk — CHANGES_REQUESTED** [review]
See above.

**Pull request merged**